server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestGetMappingAction.java

```diff
@@ -83,6 +84,7 @@ public RestChannelConsumer prepareRequest(final RestRequest request, final NodeC
         final GetMappingsRequest getMappingsRequest = new GetMappingsRequest();
```

Comment:
we may also want to look at removing RestGetAllMappingsAction if possible. As a follow-up.